bin/moodle-docker-compose.cmd

```diff
@@ -64,6 +64,10 @@ IF NOT "%MOODLE_DOCKER_PHPUNIT_EXTERNAL_SERVICES%"=="" (
     SET DOCKERCOMPOSE=%DOCKERCOMPOSE% -f "%BASEDIR%\phpunit-external-services.yml"
 )
 
+IF NOT "%MOODLE_DOCKER_BEHAT_FAILDUMP%"=="" (
```

Comment:
Hi @aspark21,
same as above, what about adding a dir check even here?
Kind of `exist "%MOODLE_DOCKER_BEHAT_FAILDUMP%\nul"`.

HTH,
Matteo

Reply:
Aligned the checks with those done for MOODLE_DOCKER_WWWROOT but nested as this is not a mandatory env variable